docker-compose.yml

```diff
@@ -1,7 +1,23 @@
 version: '3.4'
 
 services:
+  db:
+    build: ./mysql
+    environment:
+      MYSQL_ROOT_PASSWORD: root
+      MYSQL_DATABASE: govwifi_test
+    ports:
+      - "13306:3306"
+
   app:
     build: .
+    environment:
+      DB_NAME: govwifi_test
+      DB_PASS: root
+      DB_USER: root
+      DB_HOSTNAME: db
+      FRONTEND_API_KEY: supersecret
+    links:
+      - db
     ports:
       - "8080:8080"
```

Comment:
Unrelated to this PR, but looks like we'll have some trouble in the near future with port conflicts locally.

Reply:
Cross that bridge when we come to it I guess